nautobot/extras/datasources/git.py

```diff
@@ -176,6 +236,22 @@ def ensure_git_repository(repository_record, job_result=None, logger=None, head=
     to_path = repository_record.filesystem_path
     from_branch = repository_record.branch
 
+    return from_url, to_path, from_branch
```

Comment:
Per the other comment about tuples, please return a `collections.namedtuple` here instead of a naked tuple!